internal/permissiongroups/stories/stories.go

```diff
@@ -0,0 +1,36 @@
+package storypermissiongroups
+
+import (
+	"github.com/source-academy/stories-backend/internal/auth/permissions"
+	userpermissions "github.com/source-academy/stories-backend/internal/auth/permissions/users"
+)
+
+func Create() permissions.PermissionGroup {
+	return userpermissions.
+		GetRolePermission(userpermissions.CanCreateStories)
+}
+
+func Read() permissions.PermissionGroup {
+	return userpermissions.
+		GetRolePermission(userpermissions.CanReadStories)
```

Comment:
If i understand correctly, there should be a group check here when the group is set up right?